sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -215,22 +217,26 @@ private Transforms() {
      * @return The list of per page {@code FormTable}.
      */
     static List<FormTable> getPageTables(PageResult pageResultItem, List<ReadResult> readResults, int pageNumber) {
-        return pageResultItem.getTables().stream()
-            .map(dataTable ->
-                new FormTable(dataTable.getRows(), dataTable.getColumns(),
-                    dataTable.getCells()
-                        .stream()
-                        .map(dataTableCell -> new FormTableCell(
-                            dataTableCell.getRowIndex(), dataTableCell.getColumnIndex(),
-                            dataTableCell.getRowSpan() == null ? DEFAULT_TABLE_SPAN : dataTableCell.getRowSpan(),
-                            dataTableCell.getColumnSpan() == null ? DEFAULT_TABLE_SPAN : dataTableCell.getColumnSpan(),
-                            dataTableCell.getText(), toBoundingBox(dataTableCell.getBoundingBox()),
-                            dataTableCell.getConfidence(),
-                            dataTableCell.isHeader() == null ? false : dataTableCell.isHeader(),
-                            dataTableCell.isFooter() == null ? false : dataTableCell.isFooter(),
-                            pageNumber, setReferenceElements(dataTableCell.getElements(), readResults)))
-                        .collect(Collectors.toList()), pageNumber))
-            .collect(Collectors.toList());
+        if (pageResultItem.getTables() == null) {
```

Comment:
just curious, what drove this change?

Reply:
Invoice recognition is returning pageResults with null tables. So needed to add this to avoid NPE.

Reply:
ohh yes that is right. Thank you :)